test/test_epoll.cpp

```diff
@@ -754,8 +753,8 @@ class TestEPoll: public srt::Test
 
 TEST_F(TestEPoll, SimpleAsync)
 {
-    SRTSOCKET ss = SRT_INVALID_SOCK;
-    createServerSocket( (ss) );
+    srt::UniqueSocket ss;
+    createServerSocket( (ss.ref()) );
```

Comment:
What if via move semantics?
```suggestion
    srt::UniqueSocket ss = createServerSocket( (ss.ref()) );
```

Reply:
Actually I don't know why there was a socket passed by reference. I think in such solution there's not required passing anything.

Reply:
Ah sorry. This is not possible. If you have any other type than void in the methods in the text fixture, you can't use ASSERT_EQ in the area of this method.